regression/cbmc-java/json_trace3/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+Test.class
+--cover location --trace --json-ui --unwind 5
```

Comment:
We generally try to use the same options as no platform, but it is not a strict rule.